package.json

```diff
@@ -154,12 +154,12 @@
   },
   "scripts": {
     "ci-lint": "eslint --format github .",
-    "ci-lint-files": "eslint --format github",
     "ci-test": "nyc npm run test",
-    "lint": "eslint .",
-    "lint-fix": "eslint --fix .",
+    "lint": "eslint . --fix",
+    "lint-check": "eslint .",
+    "lint-files": "eslint",
```

Comment:
I'm assuming ci-lint-files was a typo, changed to `lint-files` to be in line with the lint/format tasks

Reply:
Where do you say the typo was? The ci- prefix was because only the ci called this, but happy to have more uniform naming :)

Reply:
Assumed it was a typo because ci-x tasks are ran on CI but there's no staged files in CI (because they're commited)